pkg/cmd/template/data_values_flags.go

```diff
@@ -55,6 +56,7 @@ func (s *DataValuesFlags) Set(cmdFlags CmdFlags) {
 	cmdFlags.StringArrayVar(&s.FromFiles, "data-values-file", nil, "Set multiple data values via a YAML file (format: /file/path.yml) (can be specified multiple times)")
 
 	cmdFlags.BoolVar(&s.Inspect, "data-values-inspect", false, "Calculate the final data values (applying any overlays) and display that result")
+	cmdFlags.BoolVar(&s.SkipValidation, "data-values-disable-validation", false, "Do not attempt to validate data values (warning: might allow invalid values)")
```

Comment:
data-values-disable-validation => data-values-validation=true (enabled by default)

should this be called `data-values-schema-validation`? is validation part of schema or can i also apply validations without schema?

Reply:
> data-values-disable-validation => data-values-validation=true (enabled by default)

👍🏻  (note conversation above)

Reply:
> should this be called data-values-schema-validation? is validation part of schema or can i also apply validations without schema?

This flag was meant to short-circuit validating data values, regardless of the source of the validation rules.

And yes, you can annotate `@assert/validate` in the case where you want to write validations without having to specify Schema.